### PR TITLE
Correct distfile names for various ebuilds

### PR DESCRIPTION
**Pull request by kholk**

### CODE REVIEW

**Comment by Amynka** [comment]
@mgorny

File: dev-lang/lfe/lfe-0.10.1.ebuild [hunk 1]
```diff
@@ -7,7 +7,7 @@ inherit multilib
 
 DESCRIPTION="Lisp-flavoured Erlang"
 HOMEPAGE="http://lfe.github.io/"
-SRC_URI="https://github.com/rvirding/lfe/archive/v${PV}.zip"
+SRC_URI="https://github.com/rvirding/lfe/archive/v${PV}.zip -> ${P}.tar.gz"
```
Comment (Polynomial-C) [inline dev-lang/lfe/lfe-0.10.1.ebuild]: Eh... you rename a zip file to a gzipped tarball? Why?

File: dev-libs/hyperscan/Manifest [hunk 2]
```diff
@@ -1 +1,2 @@
+DIST hyperscan-4.4.1.tar.gz 1515106 SHA256 3a082d92a3cb0cd724bc1190d24cc39752bd3db35d22115fda03d2e91ccd94cd SHA512 5e3463061110f81ad145dccf566f8308eed7bfd6711af59461b59903dd77b20963df3dd0f0ec8fc925837e76b658352b420b1ce210c4c91a4cb128a4d5f83056 WHIRLPOOL 1eff40644b4284615f8011e4fd516005381468d651a2675c0e4b46b2b34a9986a5c242590314ef04ebe8528e6ee9090c07830d868de6fed84a0c244ad5ebf28d
 DIST v4.4.1.tar.gz 1515106 SHA256 3a082d92a3cb0cd724bc1190d24cc39752bd3db35d22115fda03d2e91ccd94cd SHA512 5e3463061110f81ad145dccf566f8308eed7bfd6711af59461b59903dd77b20963df3dd0f0ec8fc925837e76b658352b420b1ce210c4c91a4cb128a4d5f83056 WHIRLPOOL 1eff40644b4284615f8011e4fd516005381468d651a2675c0e4b46b2b34a9986a5c242590314ef04ebe8528e6ee9090c07830d868de6fed84a0c244ad5ebf28d
```
Comment (Polynomial-C) [inline dev-libs/hyperscan/Manifest]: Why is the distfile without package name still in this Manifest? Is there a hyperscan-4.4.1.ebuild which also needs adjustment?

File: dev-lang/lfe/lfe-0.10.1.ebuild (hunk 1 above)
Comment (kholk) [inline dev-lang/lfe/lfe-0.10.1.ebuild]: whooooops! sorry, that was not intentional, got distracted.

File: dev-libs/hyperscan/Manifest (hunk 2 above)
Comment (kholk) [inline dev-libs/hyperscan/Manifest]: Yes, there is.

**Comment by kholk** [comment]
dev-lang/lfe problem fixed.

File: dev-libs/hyperscan/Manifest (hunk 2 above)
Comment (Polynomial-C) [inline dev-libs/hyperscan/Manifest]: Would you mind adjusting that ebuild as well?

Comment (kholk) [inline dev-libs/hyperscan/Manifest]: Sure, no problem.

**Comment by kholk** [comment]
hyperscan ebuild changed as per @Polynomial-C request

File: dev-lang/lfe/Manifest [hunk 3]
```diff
@@ -1,4 +1,4 @@
-DIST v0.10.1.zip 296228 SHA256 05de4cd5a15ea37e505049c5d7e9c75fbb61ce1bcabec39d762df3dbeda7d814 SHA512 94719b42ecf6dc5505ea649df00b255b1e901d6104426192f34692e2f7bc85940b96b39fd1bfa00577b63c17e10d8fd51d3de743c02d8ac2263abe2fb47d4498 WHIRLPOOL a3c4420421afd45bf6a0908a3a0f42319a094d4d186636eebe5f58b4f2d11192c0a7cbc6eee7dbc1aa48f326fed8c1869932e4d277c41387f790006675f4985d
+DIST lfe-0.10.1.zip 296228 SHA256 05de4cd5a15ea37e505049c5d7e9c75fbb61ce1bcabec39d762df3dbeda7d814 SHA512 94719b42ecf6dc5505ea649df00b255b1e901d6104426192f34692e2f7bc85940b96b39fd1bfa00577b63c17e10d8fd51d3de743c02d8ac2263abe2fb47d4498 WHIRLPOOL a3c4420421afd45bf6a0908a3a0f42319a094d4d186636eebe5f58b4f2d11192c0a7cbc6eee7dbc1aa48f326fed8c1869932e4d277c41387f790006675f4985d
 DIST v0.7a.zip 216133 SHA256 c494135bf50a8dd62723e026a6b4cf4c5c4d6b8a01e59b6e41bbb6daf5c25dd3 SHA512 93f1b0e81275af21d3dbee73f364f4b67d4ce2d6beb7fe2c17e17d5f5330ec8762d63978ae291267ddd92d311b2ad7d2572854b5e3da9c884f4c706e23551aaa WHIRLPOOL 44067ac8e529d54e66cb18d0ae11c3f7d3a489b530f995e90eb000aae9e212ce79abc57388c515a681265a0242f568ae7efb3e1307e937c3aad55438b6d7b0da
```
Comment (Dessa) [inline dev-lang/lfe/Manifest]: looks like the others were forgotten

**Pull request closed without merging**